apps/communications/contacts/js/views/form.js

```diff
@@ -822,6 +854,12 @@ contacts.Form = (function() {
       formHeader.addEventListener('action', cancelHandler);
       doMatch(contact, callbacks);
     });
+    var title = formTitle.getAttribute('data-l10n-id');
+    if (title === 'editContact') {
+      Contacts.showStatus({
+        id: 'updateContact'
+      });
+    }
```

Comment:
It would be better to use another data attribute not l10n-id.
This attribute is used for localization only.